--- charts-original/templates/_helpers.tpl
+++ charts/templates/_helpers.tpl
@@ -75,3 +75,16 @@
 {{- define "remotedialer-proxy.rolebinding" -}}
 {{- include "api-extension.name" . }}
 {{- end }}
+
+{{/*
+Image pull secrets
+*/}}
+{{- define "imagePullSecrets" -}}
+{{- if .Values.global.cattle.imagePullSecrets }}
+imagePullSecrets:
+  {{- range .Values.global.cattle.imagePullSecrets }}
+  - name: {{ . }}
+  {{- end }}
+{{- end }}
+{{- end -}}
+
